### ISSUE TITLE
Ability to share a filter state of the React app via URL

### ISSUE THREAD

**Issue by frjo** [OpenTechFund/hypha#2248]
Related #2203

**Comment by danblah** [comment]
Fail, this is what i see when click the link button or save the URL and paste it:
![Screenshot 2021-03-02 at 14 20 10](https://user-images.githubusercontent.com/620336/109702658-826b1780-7b62-11eb-90c5-4470955051fd.png)

**Comment by vimal1083** [comment]
@danblah 

I guess you have taken this screen shot before adding summary to these submissions. If a submission has summary it will be displayed beneath submission name. 

Check the below URL, It has 6 submission ids out of these 2 has summary, so those 2 summaries are being displayed.
https://test-apply.opentech.fund/apply/submissions/grouped-applications/?id=9,18,22,23,27,32

<img width="1599" alt="Screen Shot 2021-03-03 at 12 32 56" src="https://user-images.githubusercontent.com/8245654/109766771-a3903e80-7c1c-11eb-8fe6-4ae0fd3f54fc.png">

**Comment by vimal1083** [comment]
I thought this requirement is same as grouped submission sharable link. Now I understand we need a way to share the filter state via URL for react app. Will start to work on this today.

**Comment by Parbhat** [comment]
The button to open grouped submissions is misleading here. The user thinks that after applying filters the share button will copy the URL to the clipboard so the current state of React app with applied filters can be shared with other users. We should add a different button to show grouped summary list of submissions. The share button should copy the URL of current state of React view with filters once we have the ability to have a separate URL for it. It will be separate PR from the #2249 


@vimal1083 Let's do the following

1. Change the icon and text on hover to highlight that this button will open grouped submissions.
2. Create separate PR for this issue.

<img width="484" alt="Screen Shot 2021-03-03 at 2 59 20 PM" src="https://user-images.githubusercontent.com/4817493/109784727-5ec2d280-7c31-11eb-8e18-49dde6189c6d.png">

**Comment by frjo** [comment]
Good idea to have this as a separate PR and remove it from https://github.com/OpenTechFund/hypha/pull/2249.

@vimal1083 Can we have the filters in the URL for React as we have in Django, that would make it easy to share a filter state?

**Comment by vimal1083** [comment]
@frjo 
yes, we can have filters in the URL for React. Will create separate PR for this

**Comment by Parbhat** [comment]
Development is completed in a branch, will create PR once #2249 is merged.
